### PR TITLE
Mac wip

### PR DESCRIPTION
**Pull request by maiselkelly**

### CODE REVIEW

File: src/main/resources/templates/ChatHomePage.html [hunk 1]
```diff
@@ -1,19 +1,24 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.w3.org/1999/xhtml">
+<html xmlns:th="http://www.w3.org/1999/xhtml" lang="en">
 <head>
     <title>Roomiez :: Searching by Online</title>
-    <link rel="stylesheet" href="/Images/stylesheet/style.css">
```
Comment (maiselkelly) [inline src/main/resources/templates/ChatHomePage.html:5]: Do not need "../"

File: src/main/resources/templates/ChatHomePage.html [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-
+    <link rel="stylesheet" href="../Images/stylesheet/style.css">
 </head>
 
 <div class="header">
 
-    <a href="/home"><img class="roomiez" src="/Images/images/Roomiez.png"/></a>
+
+    `<a href="/home"><img class="roomiez" src="../Images/images/Roomiez.png" alt="Roomiez Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/ChatHomePage.html:11]: Do not need "../"

File: src/main/resources/templates/ChatHomePage.html [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/ChatHomePage.html:21]: Do not need "../"

File: src/main/resources/templates/ChatHomePage.html [hunk 4]
```diff
@@ -22,17 +27,20 @@
...
+        <form th:action="@{/logout}" method="post">
+            <input type="submit" value="Log Out"></form>
+        /li>
 </ul>
 
 
 <div class="registrationcontainer">
 
-<img class="chatlogo" src="/Images/images/RoomiezChat.png"/>
+    <img class="chatlogo" src="../Images/images/RoomiezChat.png" alt="Chat logo"/>
```
Comment (maiselkelly) [inline src/main/resources/templates/ChatHomePage.html:43]: Do not need "../"

File: src/main/resources/templates/HomePage.html [hunk 5]
```diff
@@ -1,49 +1,48 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <title>Roomiez</title>
-    <link rel="stylesheet" href="/Images/stylesheet/style.css">
-
+    <link rel="stylesheet" href="../Images/stylesheet/style.css">
```
Comment (maiselkelly) [inline src/main/resources/templates/HomePage.html:5]: Do not need "../"

File: src/main/resources/templates/HomePage.html [hunk 6]
```diff
@@ -1,49 +1,48 @@
...
-            <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+    <div class="links">
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
 
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
 
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/HomePage.html:21]: Do not need "../"

File: src/main/resources/templates/HomePage.html [hunk 7]
```diff
@@ -1,49 +1,48 @@
...
 
     </div>
 </div>
 <body>
-
+<hr>
```
Comment (maiselkelly) [inline src/main/resources/templates/HomePage.html:26]: Do not need Line here

File: src/main/resources/templates/HomePage.html [hunk 8]
```diff
@@ -1,49 +1,48 @@
...
 
 
-<div class="alignbutton">
-<form method="get" action="/Registration">
-    <input type="submit" class="button" id="signupbutton" value="Join Today!" align="center">
-</form>
-</div>
+    <div class="alignbutton">
+        <form method="get" action="/Registration">
+            <input type="submit" class="button" id="signupbutton" value="Join Today!">
```
Comment (maiselkelly) [inline src/main/resources/templates/HomePage.html:43]: Need it to be aligned to center

File: src/main/resources/templates/HomeTour.html [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+    <div class="links">
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/HomeTour.html:22]: Do not need "../"

File: src/main/resources/templates/LogIn.html [hunk 10]
```diff
@@ -1,26 +1,25 @@
...
-    <link rel="stylesheet" href="/Images/stylesheet/style.css">
-
+    <link rel="stylesheet" href="../Images/stylesheet/style.css">
 </head>
 
 <div class="header">
 
 
-    <a href="/home"><img class="roomiez" src="/Images/images/Roomiez.png"/></a>
+    `<a href="/home"><img class="roomiez" src="../Images/images/Roomiez.png" alt="Roomiez Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/LogIn.html:12]: Do not need "../"

File: src/main/resources/templates/LogIn.html [hunk 11]
```diff
@@ -1,26 +1,25 @@
...
-
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
 
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
 
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/LogIn.html:22]: Do not need "../"

File: src/main/resources/templates/LogIn.html [hunk 12]
```diff
@@ -33,33 +32,35 @@
     <h1>LOG IN TO FIND YOUR POTENTIAL ROOMIEZ</h1>
     <br>
     <br>
-    <div class="registrationcontainer">
+    < class="registrationcontainer">
```
Comment (maiselkelly) [inline src/main/resources/templates/LogIn.html:35]: needs a div

File: src/main/resources/templates/PotentialMatches.html [hunk 13]
```diff
@@ -1,25 +1,24 @@
-
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
     <title>Roomiez :: Searching for Results</title>
-    <link rel="stylesheet" href="/Images/stylesheet/style.css">
-
+    <link rel="stylesheet" href="../Images/stylesheet/style.css">
```
Comment (maiselkelly) [inline src/main/resources/templates/PotentialMatches.html:5]: Do not need "../"

File: src/main/resources/templates/PotentialMatches.html [hunk 14]
```diff
@@ -1,25 +1,24 @@
...
-
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
 
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
 
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/PotentialMatches.html:21]: Do not need "../"

File: src/main/resources/templates/PotentialMatches.html [hunk 15]
```diff
@@ -28,22 +27,25 @@
...
-    <a href="/SearchOnline"><img class="searchbuttons1" src="/Images/images/online.png"/></a>
-    <a href="/SearchRecommended"><img class="searchbuttons" src="/Images/images/recommended.png"/></a>
+    <a href="/templates/SearchLocation.html"><img class="searchbuttons1" src="../Images/images/location.png"
+                                                  alt="Location image logo"/></a>
+    <a href="/templates/SearchNewMember.html"><img class="searchbuttons" src="../Images/images/newmember.png"
+                                                   alt="New member image logo"/></a>
+    <a href="/templates/SearchOnline.html"><img class="searchbuttons1" src="../Images/images/online.png"
+                                                alt="online sign image logo"/></a>
+    <a href="/templates/SearchRecommended.html"><img class="searchbuttons" src="../Images/images/recommended.png"
+                                                     alt="recommended image logo"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/PotentialMatches.html:46]: Do not need "../"

File: src/main/resources/templates/Registration.html [hunk 16]
```diff
@@ -1,24 +1,24 @@
 <!DOCTYPE html>
-<html xmlns:th="http://thymeleaf.org">
```
Comment (maiselkelly) [inline src/main/resources/templates/Registration.html:2]: thymeleaf is fine

File: src/main/resources/templates/Registration.html [hunk 17]
```diff
@@ -1,24 +1,24 @@
...
-
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
 
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
 
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/Registration.html:21]: Do not need "../"

File: src/main/resources/templates/Registration.html [hunk 18]
```diff
@@ -34,23 +34,36 @@ <h1>CREATE AN ACCOUNT</h1>
...
-            <input type="radio" name="gender" value="other">Prefer not to say
+            <label>
+                <input type="radio" name="gender" value="male">
+            </label>Male
+            <label>
+                <input type="radio" name="gender" value="female">
+            </label>Female
+            <label>
+                <input type="radio" name="gender" value="other">
+            </label>Prefer not to say
```
Comment (maiselkelly) [inline src/main/resources/templates/Registration.html:66]: Formatting doesn't need to be changed

File: src/main/resources/templates/SearchLocation.html [hunk 19]
```diff
@@ -1,19 +1,24 @@
...
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchLocation.html:21]: Do not need "../"

File: src/main/resources/templates/SearchLocation.html [hunk 20]
```diff
@@ -22,11 +27,11 @@
...
-    <li><a href="/UserHomePage"> Home Page</a></li>
-    <li><a href="/SearchByLocation">Search by Location</a></li>
-    <li><a href="/SearchNewMembers">Search by New Members</a></li>
-    <li><a href="/SearchOnline">Search by Online Members</a></li>
-    <li><a href="/SearchRecommended">Search by Recommended Users</a></li>
+    <li><a href="/templates/UserHomePage.html"> Home Page</a></li>
+    <li><a href="/templates/SearchLocation.html">Search by Location</a></li>
+    <li><a href="/templates/SearchNewMember.html">Search by New Members</a></li>
+    <li><a href="/templates/SearchOnline.html">Search by Online Members</a></li>
+    <li><a href="/templates/SearchRecommended.html">Search by Recommended Users</a></li>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchLocation.html:34]: No /templates

File: src/main/resources/templates/SearchNewMember.html [hunk 21]
```diff
@@ -1,19 +1,25 @@
...
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchNewMember.html:22]: Do not need "../"

File: src/main/resources/templates/SearchOnline.html [hunk 22]
```diff
@@ -1,19 +1,23 @@
...
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchOnline.html:20]: Do not need "../"

File: src/main/resources/templates/SearchOnline.html [hunk 23]
```diff
@@ -22,12 +26,12 @@
...
-    <li><a href="/SearchNewMembers">Search by New Members</a></li>
-    <li><a href="/SearchOnline">Search by Online Members</a></li>
-    <li><a href="/SearchRecommended">Search by Recommended Users</a></li>
-    <li style="float:right"><a href="/logOut">Log Out</a></li>
+    <li><a href="/templates/UserHomePage.html"> Home Page</a></li>
+    <li><a href="/templates/SearchLocation.html">Search by Location</a></li>
+    <li><a href="/templates/SearchNewMember.html">Search by New Members</a></li>
+    <li><a href="/templates/SearchOnline.html">Search by Online Members</a></li>
+    <li><a href="/templates/SearchRecommended.html">Search by Recommended Users</a></li>
+    <li style="float:right"><a href="/logout">Log Out</a></li>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchOnline.html:34]: no /templates

File: src/main/resources/templates/SearchRecommended.html [hunk 24]
```diff
@@ -1,19 +1,23 @@
...
-    <a href="/home"><img class="roomiez" src="/Images/images/Roomiez.png"/></a>
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
 
-    <div class="links">
-        <a href="https://www.facebook.com"><img class="facebook" src="/Images/images/facebook.png"/></a>
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchRecommended.html:20]: Do not need "../"

File: src/main/resources/templates/SearchRecommended.html [hunk 25]
```diff
@@ -23,12 +27,12 @@
...
-   <li><a href="/SearchNewMembers">Search by New Members</a></li>
-   <li><a href="/SearchOnline">Search by Online Members</a></li>
-    <li><a href="/SearchRecommended">Search by Recommended Users</a></li>
-   <li style="float:right"><a href="/logOut">Log Out</a></li>
+    <li><a href="/templates/UserHomePage.html"> Home Page</a></li>
+    <li><a href="/templates/SearchLocation.html">Search by Location</a></li>
+    <li><a href="/templates/SearchNewMember.html">Search by New Members</a></li>
+    <li><a href="/templates/SearchOnline.html">Search by Online Members</a></li>
+    <li><a href="/templates/SearchRecommended.html">Search by Recommended Users</a></li>
+    <li style="float:right"><a href="/logout">Log Out</a></li>
```
Comment (maiselkelly) [inline src/main/resources/templates/SearchRecommended.html:35]: no /templates

File: src/main/resources/templates/Settings.html [hunk 26]
```diff
@@ -1,19 +1,24 @@
...
-        <a href="https://www.facebook.com"><img class="facebook" src="/Images/images/facebook.png"/></a>
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
```
Comment (maiselkelly) [inline src/main/resources/templates/Settings.html:20]: Do not need "../"

File: src/main/resources/templates/Settings.html [hunk 27]
```diff
@@ -22,11 +27,14 @@
...
-    <li><a href="/ChatHomePage">Chat with Roomiez</a></li>
-    <li style="float:right"><form th:action="@{/logout}" method="post"><input type="submit" value="Log Out"/></li>
+    <li><a href="/templates/UserHomePage.html">[[${#httpServletRequest.remoteUser}]]'s Home Page</a></li>
+    <li><a href="/templates/UpdateUserProfile.html">Edit Profile</a></li>
+    <li><a href="/templates/PotentialMatches.html">Search for Matches</a></li>
+    <li><a href="/templates/ChatHomePage.html">Chat with Roomiez</a></li>
+
+    <li style="float:right"><form th:action="@{/logout}" method="post">
+        <input type="submit" value="Log Out"></form>
+    </li>
```
Comment (maiselkelly) [inline src/main/resources/templates/Settings.html:37]: no /templates

File: src/main/resources/templates/UpdateUserProfile.html [hunk 28]
```diff
@@ -1,19 +1,24 @@
...
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
+                                                 alt="Instagram Logo picture"/></a>
```
Comment (maiselkelly) [inline src/main/resources/templates/UpdateUserProfile.html:21]: Do not need "../"

File: src/main/resources/templates/UpdateUserProfile.html [hunk 29]
```diff
@@ -22,11 +27,16 @@
...
+    <li><a href="/templates/UserHomePage.html">[[${#httpServletRequest.remoteUser}]]'s Home Page</a></li>
+    <li><a href="/templates/PotentialMatches.html">Search for Matches</a></li>
+    <li><a href="/templates/ChatHomePage.html">Chat with Roomiez</a></li>
+    <li><a href="/templates/Settings.html">Settings</a></li>
+    <li><a href="/templates/HomePage.html">Home Tour</a></li>
+
+    <li style="float:right">
+        <form th:action="@{/logout}" method="post">
+            <input type="submit" value="Log Out"></form>
+    </li>
```
Comment (maiselkelly) [inline src/main/resources/templates/UpdateUserProfile.html:39]: no /templates

File: src/main/resources/templates/UserHomePage.html [hunk 30]
```diff
@@ -3,42 +3,48 @@
...
-        <a href="https://www.facebook.com"><img class="facebook" src="/Images/images/facebook.png"/></a>
-        <a href="https://www.twitter.com"><img class="twitter" src="/Images/images/twitter.png"/></a>
-        <a href="https://www.instagram.com"><img class="insta" src="/Images/images/insta.jpeg"/></a>
+        <a href="https://www.facebook.com"><img class="facebook" src="../Images/images/facebook.png"
+                                                alt="Facebook Logo picture"/></a>
+
+        <a href="https://www.twitter.com"><img class="twitter" src="../Images/images/twitter.png"
+                                               alt="Twitter Logo picture"/></a>
+
+        <a href="https://www.instagram.com"><img class="insta" src="../Images/images/insta.jpeg"
```
Comment (maiselkelly) [inline src/main/resources/templates/UserHomePage.html:21]: Do not need "../"

File: src/main/resources/templates/UserHomePage.html [hunk 31]
```diff
@@ -3,42 +3,48 @@
...
-    <li style="float:right"><form th:action="@{/logout}" method="post"><input type="submit" value="Log Out"/></li>
+    <li><a href="/templates/UpdateUserProfile.html">Edit Profile</a></li>
+    <li><a href="/templates/PotentialMatches.html">Search for Matches</a></li>
+    <li><a href="/templates/ChatHomePage.html">Chat with Roomiez</a></li>
+    <li><a href="/templates/Settings.html">Settings</a></li>
+
+    <li style="float:right">
+        <form th:action="@{/logout}" method="post">
+            <input type="submit" value="Log Out"></form>
+    </li>
```
Comment (maiselkelly) [inline src/main/resources/templates/UserHomePage.html:38]: no /templates

**Comment by UltimateStarCoder** [comment]
All this code and my changes will be discarded

**Pull request closed without merging**